resources/util.go

```diff
@@ -270,3 +270,27 @@ func GetGID(group string) (int, error) {
 
 	return -1, errwrap.Wrapf(err, "group lookup error (%s)", group)
 }
+
+// strSliceCompare takes two lists of strings and returns whether or not they
+// are equivalent. It will return nil if both sets contain the same elements,
+// regardless of order, and an error if they do not.
+func strSliceCompare(a, b []string) error {
```

Comment:
This should be public please. Do you mind renaming to strSortedSliceCompare to be more clear that it sorts?